pontoon/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "server",
+  "private": true,
+  "dependencies": {
+    "terser": "^5.10.0",
+    "yuglify": "^2.0.0"
+  }
+}
```

Comment:
I remember something flying by that might be related to this, but is this still needed?

If so, is this documented? I haven't read all the docs, I'm afraid

Reply:
These are required for django-pipeline, which needs the executables to be available somewhere.

Reply:
Right:
https://github.com/eemeli/pontoon/blob/rollup-all/pontoon/settings/base.py#L533

Axel you might remember either of these:
https://bugzilla.mozilla.org/show_bug.cgi?id=1687371
https://bugzilla.mozilla.org/show_bug.cgi?id=1692423

Reply:
I'm thrown off because there's no `pontoon/node_modules`. But I also don't understand workspaces.

In any way, though, should these be `devDependencies`?